source/tit/core/sys/utils.cpp

```
@@ -94,13 +94,13 @@ auto tty_width(TTY tty) -> std::optional<size_t> {
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-auto try_demangle(const char* mangled_name) -> std::optional<std::string> {
-  const boost::core::scoped_demangled_name demangled_name{mangled_name};
+auto try_demangle(CStrView mangled_name) -> std::optional<std::string> {
+  const boost::core::scoped_demangled_name demangled_name{mangled_name.c_str()};
   if (const auto* p = demangled_name.get(); p != nullptr) return p;
   return std::nullopt;
 }
 
-auto maybe_demangle(const char* mangled_name) -> std::string {
+auto maybe_demangle(CStrView mangled_name) -> std::string {
```

cppcheck (2.10-2)

[style] The function 'maybe_demangle' is never used.

(unusedFunction)